Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -175,6 +175,31 @@ + (BOOL)cameraIsPresent {
     return [AVCaptureDevice defaultDeviceWithMediaType:AVMediaTypeVideo] != nil;
 }
 
++ (BOOL)hasCamera:(MTBCamera)camera {
+    AVCaptureDevicePosition position = [self devicePositionForCamera:camera];
+
+    for (AVCaptureDevice *device in [AVCaptureDevice devicesWithMediaType:AVMediaTypeVideo]) {
+        if (device.position == position) {
+            return YES;
+        }
+    }
+
+    return NO;
+}
+
++ (MTBCamera)oppositeCameraOf:(MTBCamera)camera {
```

Comment:
Is `opposite` the best way to phrase it? Or will it fall apart for iPhone 7 Plus cams or the *next big thing*?